integration/integration_test.go

```diff
@@ -6160,6 +6160,19 @@ func testList(t *testing.T, suite *integrationTestSuite) {
 	}
 }
 
+func containsSubsequence(s, subseq string) bool {
```

Comment:
Can you add a godoc to this? It's not clear to me how it differs from a standard `strings.Contains`.